### PR TITLE
IDE-579 Add Crtl+N to the Repository Tree Tab to create new attribute

### PR DESCRIPTION
**Pull request by dehilsterlexis**
IDE-579 Add Crtl+N to the Repository Tree Tab to create new attribute

Fixes IDE-579

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith finally got things straight! Here is the branch you were looking for and it is up to date. Careful: I may just get this git thing... LOL.

**Comment by GordonSmith** [comment]
Thx - in general no need to do the whole PR thang, the "IDE-579" branch is all I needed.

File: wlib/RepositoryView.h [hunk 1]
```diff
@@ -435,6 +435,21 @@ class CRepositoryViewT
...
+        CString label;
+        CComPtr<IAttributeType> type;
+        m_Root->GetModules();
+        /*
+        if (GetInsertAttribute(s.mods[0]->GetQualifiedLabel(), label, type, true))
+        {
+            if (pT->m_view.DoInsertAttribute(s.mods[0].get(), (const TCHAR *)label, type))
+                GetIMainFrame()->OpenAttribute(label, type, s.mods[0].get()->GetQualifiedLabel());
+        }
+        */
```
Comment (GordonSmith) [inline wlib/RepositoryView.h:449]: Remove commented code.

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by dehilsterlexis** [comment]
@richardkchapman @GordonSmith this needs to be closed. It is old and I put it in the wrong place. The new IDE-579 is in the correct branch which I just pushed.

**Comment by richardkchapman** [comment]
As this is a PR against your own repository, I think only you can close it

**Comment by dehilsterlexis** [comment]
@richardkchapman you are right. Closed.

**Pull request closed without merging**